ledger/participant-integration-api/src/main/scala/platform/store/FlywayMigrations.scala

```diff
@@ -53,8 +53,9 @@ private[daml] class FlywayMigrations(
   def migrate(allowExistingSchema: Boolean = false): Future[Unit] = run { configBase =>
     val flyway = configBase
       .baselineOnMigrate(allowExistingSchema)
-      .baselineVersion(MigrationVersion.fromVersion("0"))
+      .baselineVersion(MigrationVersion.fromVersion("38.1"))
       .ignoreFutureMigrations(false)
+      .ignoreMigrationPatterns("*:missing", "*:ignored")
```

Comment:
Ignoring `missing`, because we deleted migrations v1 to v38 in the application.

Ignoring `ignored`, because we added migration v38.1 in the application, in the middle of the migration history.

Reply:
Does it mean you're disabling validation for `missing` and `ignored` migrations so that if we erroneously have a `missing` or `ignored` migration in the future then flyway won't catch it?

Reply:
I am concerned relaxing validation for all 38> and future migrations. Can we somehow relax validation only for the baselining part? Or maybe going full hardcore and do a three step migration:1) until 38.1 with ignoring 2)erradicate flyway tables 3) start migration with baseline (but no relaxation of validation)
Also I am utterly disappointed that the baseling migration feature is not free, that looks very appealing.